apps/web/src/hooks/use-novu.ts

```diff
@@ -25,7 +25,7 @@ export function useNovu() {
         elt.src = t;
         const before = o.getElementsByTagName(f)[0];
         before.parentNode?.insertBefore(elt, before);
-      })(window, document, WIDGET_EMEBED_PATH, 'novu', 'script');
+      })(window, document, WIDGET_EMBED_PATH, 'novu', 'script');
```

Comment:
Rollback this one please. This can affect current users settings and break their systems.